examples/kubernetes/demo-pod.yaml

```diff
@@ -1,17 +1,3 @@
-kind: PersistentVolumeClaim
```

Comment:
why delete this?

Reply:
Sorry, an artifact of manual testing stuff. Putting it back